src/components/tools/listing/Listing.js

```diff
@@ -86,6 +97,23 @@ function Listing(props) {
         selectedSearchTerm,
     ]);
 
+    const [removeTools, { status: deleteStatus }] = useMutation(deleteTools, {
+        onSuccess: () => {
+            announce({
+                text: t("toolDeleted"),
+            });
+            //reset selection to avoid stale selected state
+            setSelected([]);
+            queryCache.invalidateQueries(toolsKey);
+        },
+        onError: (e) => {
+            showErrorAnnouncer(t("toolDeleteError"), e);
+            //reset selection to avoid stale selected state
+            setSelected([]);
+            queryCache.invalidateQueries(toolsKey);
```

Comment:
Minor, but I wonder if it might be helpful to keep the selection on error so that the user can try to delete the tools again (in case they selected more than one)?

Reply:
It would be more convenient to try again, in case of transient errors; but because the UI is batching the API calls, which can only delete 1 tool per call, then it's probably best to reset the selection and reload the listing, in case some of the selection was successfully deleted 🤔 

Otherwise we'd have to figure out something tricky with the promises in order to fix the selection based on which were successful and which failed.

Reply:
Ah, forgot it's a batch call.  Good point.  Better to reset to avoid additional errors.